zhazam/Modules/Loading/LoadingViewController.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  LoadingViewController.swift
+//  zhazam
+//
+//  Created by Nurbek Ismagulov on 8/20/19.
+//  Copyright © 2019 Nurbek Ismagulov. All rights reserved.
+//
+
+import UIKit
+
+class LoadingViewController: UIViewController {
+    
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var percentageLabel: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        generateRandomPercents(from: Int.random(in: 1...4) * 10)
+    }
+    
+    func generateRandomPercents(from number: Int) {
+        if number < 100 {
+            let randomNumber = number + Int.random(in: 0...(100-number)/10) * 10
+            percentageLabel.text = randomNumber == 100 ? "\(randomNumber)." : "0\(randomNumber)."
```

Comment:
why generateRandomPercents modifies label's text?

Reply:
@danabeknar probably because it is violating the famous rule:
> Functions should do one thing. They should do it well. They should do it only.
I'll leave a reference for a solid book that might help you to better understand how to write cleaner, reliable and robust code! [Check it out](https://www.amazon.com/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882)